projects/explorer/src/app/pages/blocks/blocks.component.ts

```diff
@@ -20,27 +20,40 @@ export class BlocksComponent implements OnInit {
   defaultPageSize = this.pageSizeOptions[1];
   defaultPageNumber = 1;
 
-  pollingInterval = 30;
+  pollingInterval = 5;
```

Comment:
I think the polling interval setting of 5 seconds is fine for the block generation interval, but I concerned about the load on the Rest API.
I think it would be better to leave it at the original 30 seconds.

Reply:
I understand your concern. but I think 5 seoconds is better, Because,
- Auto loading is disabled default. so, access concentration is not so big problem.
- It is natural that the block come to the list imdiatory ( one by one, not 6 block together ), if auto loading enable. 

But, this is my opinion and expectations. 
Could you please tell me what you think about it ?

Reply:
OK, let's proceed in 5 second intervals once.